src/main/java/view/SubView.java

```diff
@@ -75,13 +94,42 @@ public static void main(String[] args) {
         JPanel views = new JPanel(cardLayout);
         application.add(views);
 
+        ViewManagerModel viewManagerModel = new ViewManagerModel();
+        new ViewManager(views, cardLayout, viewManagerModel);
+
+        SearchViewModel searchViewModel = new SearchViewModel();
+        ApplyViewModel applyViewModel = new ApplyViewModel();
+
         SubViewModel subViewModel = new SubViewModel();
-        SubViewController subViewController = new SubViewController();
+        SubViewPresenter subViewPresenter = new SubViewPresenter(searchViewModel, applyViewModel, viewManagerModel);
+
+        SubViewInputBoundary subViewInteractor = new SubViewInteractor(subViewPresenter);
+        SubViewController subViewController = new SubViewController(subViewInteractor);
         SubView subView = new SubView(subViewModel, subViewController);
 
         views.add(subView, subView.viewName);
 
-        application.pack();;
+        // add apply and search Views
+        ServerUserDataAccessObject userDataAccessObject = new ServerUserDataAccessObject(new ExistingCommonUserFactory());
+        SearchUserDataAccessInterface userSearchData = new SearchUserDataAccessInterface() {
+            @Override
+            public JSONObject basicQuery(String queryParameters, String optionalParameters) {
+                return null;
+            }
+        };
+
+        SearchView searchView = SearchUseCaseFactory.create(viewManagerModel, searchViewModel, userSearchData);
+        assert searchView != null;
+        views.add(searchView, searchView.viewName);
+        ApplyInputBoundary applyUseCaseInteractor = null;
+        ApplyController applyController = new ApplyController(applyViewModel, applyUseCaseInteractor);
```

Comment:
Consider making a SubView use case factory, which would make this alot cleaner (pushes the controller creation somewhere else)